sw/device/silicon_creator/imm_rom_ext/imm_rom_ext.c

```diff
@@ -30,11 +31,14 @@ static rom_error_t imm_rom_ext_start(void) {
 
   // Initialize Immutable ROM EXT.
   sec_mmio_next_stage_init();
+  HARDENED_RETURN_IF_ERROR(imm_rom_ext_epmp_reconfigure());
+
   // Configure UART0 as stdout.
   pinmux_init_uart0_tx();
   uart_init(kUartNCOValue);
 
   dbg_printf("IMM_ROM_EXT v0.1\r\n");
+  dbg_print_epmp();
```

Comment:
This function call (consuming ~600 bytes) was recently removed from the `rom_ext` for reducing the code size. Perhaps we should avoid calling this function from the immutable rom_ext as well.

Reply:
Removed. Thanks!